pages/00-Home.md

```diff
@@ -70,18 +24,31 @@ The hydrus system is built over the following standards and tools:
 
 Other than this, there are also various Python packages that hydrus uses. A list of these packages can be found in the [requirements.txt](https://github.com/HTTP-APIs/hydrus/blob/master/requirements.txt) file.
 
+## Hydra Smart Clients
+
+**Hydra smart client** (`hydra-python-agent`) is a Python implementation of a client based on the Hydra Community Group specifications and reference implementation [Heracles.ts](https://github.com/HydraCG/Heracles.ts). It can navigate a network of hydrus servers autonomously by reading their API documentation. It is a generic client because it can query and retrieve data from any, Hydra-aware or not, server (such as hydrus or any other Web server).
+Every **Hydra smart client** instance provides:
+- A client that can understand Hydra vocabulary and interact autonomously with a Hydra-supporting server via basic [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations on data. The contributors are actively developing a fork of `hydra-py` to create a generic [Hydra Agent](https://github.com/HTTP-APIs/hydra-python-agent). 
+
```

Comment:
The client we have right now is not a fork of hydra-py, it's a completely new tool developed by @Guttz .

Reply:
https://www.hydraecosystem.org/00-Home

Reply:
This is outdated @AwesomeChap that's why you were supposed to understand the tools first. Part of your job is to update the documentation not just copying the old.